src/engine/ExportQueryExecutionTrees.h

```diff
@@ -133,14 +120,15 @@ class ExportQueryExecutionTrees {
    *        then the query result will be obtained via `qet->getResult()`.
    * @return a 2D-Json array corresponding to the IdTable given the arguments
    */
-  static nlohmann::json idTableToQLeverJSONArray(
+  static cppcoro::generator<std::string> idTableToQLeverJSONBindingsStream(
       const QueryExecutionTree& qet, const LimitOffsetClause& limitAndOffset,
-      const QueryExecutionTree::ColumnIndicesAndTypes& columns,
-      std::shared_ptr<const Result> resultTable,
+      const QueryExecutionTree::ColumnIndicesAndTypes columns,
+      std::shared_ptr<const Result> result,
       CancellationHandle cancellationHandle);
 
   // ___________________________________________________________________________
-  static nlohmann::json constructQueryResultBindingsToQLeverJSON(
+  static cppcoro::generator<std::string>
+  constructQueryResultBindingsToQLeverJSONStream(
```

Comment:
No need to call it `...Stream`  as you have deleted the old version. So the original name suffices.